#!/usr/bin/env python# -*- coding: utf-8 -*-# Author: Yuanbin# Datetime: 20180109from __init__ import *class History:    def __init__(self, zbx_base=None):        self.zb = zbx_base    def get(self, **kwargs):        """        history:0——浮点数；1——字符型；2——日志类型；3——无符号整形；4——文本；        """        try:            obj = self.zb.json_obj('history.get', kwargs, auth=True)            result = self.zb.do_request(obj)            ret_get = result.get('result')            return ret_get        except Exception as e:            message = u'获取历史数据（Get history failure）失败 - %s ' % str(e)            traceback.print_exc(message)            return Falseif __name__ == '__main__':    kwargs = {        "output": "extend",        "history": 0,        "itemids": 157450,        "sortfield": "clock",        "sortorder": "DESC",        "limit": 10    }    zh = History('zbx_prod')    his_ret = zh.get(**kwargs)    print his_ret